cupy_backends/cuda/api/runtime.pxd

```diff
@@ -524,8 +524,13 @@ cpdef enum:
     cudaMemAdviseSetAccessedBy = 5
     cudaMemAdviseUnsetAccessedBy = 6
 
-    streamDefault = 0
-    streamNonBlocking = 1
+    # cudaStream flags
+    cudaStreamDefault = 0
+    cudaStreamNonBlocking = 1
+
+    # cudaStream handles
+    cudaStreamLegacy = 1
+    cudaStreamPerThread = 2
```

Comment:
Sorry, forgot to say that CuPy's convention for wrapping the CUDA API is to remove the prefix `cuda`. I think someone messed up in a few fields here and there (including me 😅), but this generally holds (see also the preamble in this file). 

In particular, `streamDefault` and `streamNonBlocking` have been around for a long time, and renaming them would cause backward incompatibility (they are public low-level API, and users could access them through `cupy.cuda.*`).

Reply:
Got it, sorry for missing that, I reverted changes to the CuPy standard, and also changed new names to `streamLegacy`/`streamPerThread`.